docs/source/index.rst

```diff
@@ -166,6 +163,10 @@ from normal code. If you are *not*
 using async-style programming, you do not need to know about how this
 works, but you might find the implementation interesting.
 
+For every synchronous function there is asynchronous one prefixed by ``_``, but
+the ``open`` operation does not support async operation. If you need it to open
+some file in async manner, it's better to asynchronously download it to
```

Comment:
Note that we *may* implement open_async at some point, which will not be like a normal file, supporting only streaming read/write and no seek or random access.